quarkus/tests/integration/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -412,6 +412,11 @@ public void setProperty(String key, String value) {
         setProperty(key, value, distPath.resolve("conf").resolve("keycloak.conf").toFile());
     }
 
+    @Override
+    public void removeProperty(String db) {
+        setProperty(db, null, distPath.resolve("conf").resolve("keycloak.conf").toFile());
```

Comment:
How the implementation for the `DockerKeycloakDistribution` would look like?

Reply:
The same as `setProperty` we have already have: unsupported.

Reply:
👍